src/System.Security.Cryptography.X509Certificates/tests/X509StoreTests.cs

```diff
@@ -151,6 +151,19 @@ public static void AddReadOnlyThrows()
         }
 
         [Fact]
+        public static void AddDisposedThrowsCryptographicException()
+        {
+            using (X509Store store = new X509Store(StoreName.My, StoreLocation.CurrentUser))
+            using (X509Certificate2 cert = new X509Certificate2(TestData.MsCertificate))
+            {
+                store.Open(OpenFlags.ReadWrite);
+
+                cert.Dispose();
+                Assert.ThrowsAny<CryptographicException>(() => store.Add(cert));
```

Comment:
I decided to change it anyway. I haven't used xUnit that much so I wasn't aware of the difference. Looked it up and agree that Throws is probably more correct in this case ;).